webrtc/pending-promise-after-close.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<title>Test pending promises behavior when RTCPeerConnection object is closed</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script type="text/javascript">
+  'use strict';
+
+  function test_never_resolved(test_func, testName) {
+    async_test(t => {
+      const pc1 = new RTCPeerConnection();
+      const closePc1 = () => pc1.close();
```

Comment:
Once the invalid "Operating on unclosed peer connection should succeed" test is removed, then this becomes unnecessary and all that's needed is passing the RTCPeerConnection object, which can be then call `pc.close()` directly.